src/components/PortalConfirmationModal/PortalConfirmationModal.tsx

```diff
@@ -0,0 +1,17 @@
+import { ConfirmationModal } from "@canonical/react-components";
+import type { ConfirmationModalProps } from "@canonical/react-components/dist/components/ConfirmationModal";
+import usePortal from "react-useportal";
+
+import "./_portal-confirmation-modal.scss";
+
+const PortalConfirmationModal = (props: ConfirmationModalProps) => {
```

Comment:
We use this in 4 different places and it might get used in other places in the future, so I thought it might make sense to make it its own component.

Reply:
Also, we change the `z-index` of `.p-modal`, thus, by making this its own component, the change to `.p-modal` would not influence other elements that might have this class.

Reply:
Thank you for this explanation!